apps/auth/main.py

```
@@ -1,40 +1,49 @@
-from fastapi import FastAPI, HTTPException, Response
+from fastapi import FastAPI, HTTPException, Response, Request
 import uvicorn
 import models
 from config import PASSWORD_MAX_LENGTH, PASSWORD_MIN_LENGTH, AUTH_PORT, ISSECURE
-import database as db
 from database import users
 from utils import hash_password, verify_password, create_jwt_token, decode_jwt_token
```

Ruff (0.8.2)

`utils.decode_jwt_token` imported but unused

Remove unused import: `utils.decode_jwt_token`

(F401)